README.md

```diff
@@ -93,7 +93,49 @@ It will help measure the impact of the TeachOpenCADD platform and future funding
 }
 ```
 
-## Funding
+## Acknowledgments
+
+### External resources
+
+#### Python packages
+
+- Cheminformatics and structural bioinformatics: 
+  [`rdkit`](http://rdkit.org/), 
+  [`openbabel`](http://openbabel.org/wiki/), 
+  [`mdanalysis`](https://www.mdanalysis.org/), 
+  [`biopython`](https://biopython.org/), 
+  [`biopandas`](http://rasbt.github.io/biopandas/), 
+  [`opencadd`](https://opencadd.readthedocs.io/en/latest/)
+- Data science (PyData stack): 
+  [`numpy`](https://numpy.org/), 
+  [`pandas`](https://pandas.pydata.org/), 
+  [`scikit-learn`](https://scikit-learn.org/), 
+  [`juypter`](https://jupyter.org/), 
```

Comment:
```suggestion
  [`jupyter`](https://jupyter.org/), 
```

Reply:
That's a typo, I do over and over again - thanks for spotting this!